packages/playground/src/components/select_gpu_node.vue

```diff
@@ -53,30 +60,32 @@
 <script lang="ts" setup>
 import { onMounted, type PropType, ref, watch } from "vue";
 
-import GPUNode, { type GPUNodeType, type NodeGPUCardType } from "@/utils/filter_node_with_gpu";
-import { getCardName, normalizeError } from "@/utils/helpers";
-
 import { useProfileManager } from "../stores/profile_manager";
 import { type Flist, ProjectName } from "../types";
+import GPUNode, { type GPUNodeType, type NodeGPUCardType } from "../utils/filter_node_with_gpu";
 import { getGrid } from "../utils/grid";
+import { getCardName, normalizeError } from "../utils/helpers";
 
 export interface GPUMachineFilters {
   ipv6: boolean;
   ipv4: boolean;
   wireguard: boolean;
   planetary: boolean;
-  hasGPU: boolean;
+  hasGPU?: boolean;
```

Comment:
what is the idea of making this field optional?

Reply:
It was added by mistake. will revert it.